spring-cloud-gcp-docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -267,4 +276,7 @@ Google Cloud Pub/Sub API, if different from the ones in the
 | `spring.cloud.gcp.pubsub.credentials.scopes` |
 https://developers.google.com/identity/protocols/googlescopes[OAuth2 scope] for Spring Cloud GCP
 Pub/Sub credentials | No | https://www.googleapis.com/auth/pubsub
+| `spring.cloud.gcp.pubsub.deserializationTrustedPackages[n]` |
```

Comment:
This sounds like very long configuration property.
The `trustedPackages` is pretty self-explanatory, plus the JavaDocs on the property should do the trick to determine the purpose.

Reply:
I don't have an opinion on this one. At the moment there is both the long name and the javadoc comment. @meltsufin If you don't have a problem with the shorter name I can change it back.

Reply:
Shorted name is ok by me.